src/npctalk.cpp

```diff
@@ -878,7 +878,7 @@ void game::chat()
 
     nmenu.addentry( NPC_CHAT_YELL, true, 'a', _( "Yell" ) );
     nmenu.addentry( NPC_CHAT_SENTENCE, true, 'b', _( "Yell a sentence" ) );
-    nmenu.addentry( NPC_CHAT_THINK, true, 'T', _( "Think something" ) );
+    nmenu.addentry( NPC_CHAT_MISC, true, 'T', _( "Miscellaneous action" ) );
```

Comment:
`Miscellaneous action` seems misleading, this looks like it's going to actually do something

Reply:
> `Miscellaneous action` seems misleading, this looks like it's going to actually do something

In the blurb that shows up in the textbox it says it'll have no in-game effect. What would be a good name that doesn't break the fourth wall?

Reply:
'Emote' is commonly used in games to represent fluff actions.

Reply:
> 'Emote' is commonly used in games to represent fluff actions.

PERFECT! I'll do that.